README.md

```diff
@@ -1,45 +1,27 @@
 ![remixer](https://cdn.rawgit.com/material-foundation/material-remixer/master/docs/assets/lockup_remixer_icon_horizontal_dark_small.svg)
 
-Remixer helps teams use and refine design specs by providing an abstraction for these values that is accessible and configurable from both inside and outside the app itself. This abstraction is called a variable. A variable has a key (e.g. "primaryColor"), a type (e.g. Color), and a value (e.g. red). Instead of hard-coding values, engineers can use variables to allow anyone on their team to refine them during development without even having to restart the app. Refinements can be done from inside the app itself, from a web remote controller, or through plugins for other design tools.
+Remixer allows you to easily define UI variables in your app that can be adjusted while it is running. This can help you and your team to iterate very quickly on design issues because you don't need to rebuild or even restart your app to see the changes reflected.
 
-The Remixer SDK is currently under development and will be available for iOS, Android and Web. Please review the State of Development table for status of each platform.
+Remixer is available in three platforms ([Android](http://github.com/material-foundation/material-remixer-android), [iOS](http://github.com/material-foundation/material-remixer-ios) and [Web](http://github.com/material-foundation/material-remixer-js)) and comes with a helpful [Remote Controller](http://github.com/material-foundation/material-remixer-remote-web) that lets you interact with variables from another device.
```

Comment:
"and all of them support setting up a Remote Controller that lets you change variables from a web dashboard" (or web controller). We shouldn't say from another device as that is confusing.